test/jdk/jdk/incubator/vector/Halffloat128VectorLoadStoreTests.java

```diff
@@ -0,0 +1,734 @@
+/*
+ * Copyright (c) 2018, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
The copyright should be only "2022".

Reply:
Just want to avoid the template changes for all test cases.